Helper/RecaptchaHelper.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+namespace Victoire\Widget\FormBundle\Helper;
+
+use Symfony\Component\DependencyInjection\ContainerInterface as Container;
+
+class RecaptchaHelper {
+
+    private $container;
+
+    public function __construct(Container $container) {
```

Comment:
We can inject the two parameters instead of the whole container: https://symfony.com/doc/3.4/service_container.html#service-parameters